src/utils/dialogmoteUtils.ts

```diff
@@ -1,11 +1,33 @@
 import {
   DialogmoteDTO,
   DialogmoteStatus,
+  SvarType,
 } from "@/data/dialogmote/types/dialogmoteTypes";
 
-export const isAktivtDialogMote = (dialogmote: DialogmoteDTO): boolean => {
+export const isAktivtDialogmote = (dialogmote: DialogmoteDTO): boolean => {
   return (
     dialogmote.status === DialogmoteStatus.INNKALT ||
     dialogmote.status === DialogmoteStatus.NYTT_TID_STED
   );
 };
+
+export const harMottattSvar = (dialogmote: DialogmoteDTO): boolean => {
+  const svarFraArbeidstaker =
+    dialogmote.arbeidstaker.varselList[0]?.svar?.svarType;
```

Comment:
Bra med null-check på varselList[0]-access også 👍

Reply:
Kudos til Luan for den 🙌🏼